README.md

```diff
@@ -93,6 +93,7 @@ const keys = extractFromFiles([
 
 - `marker`: The name of the internationalized string marker function. Defaults to `i18n`.
 - `keyLoc`: An integer indicating the position of the key in the arguments. Defaults to `0`. Negative numbers, e.g., `-1`, indicate a position relative to the end of the argument list.
+- `useTypeScript`: Defaults to `false`.
```

Comment:
Instead of this boolean, maybe the API can be

`parser: string` with a default to `flow`?

This way it's extensible.

Of course this will need a throw if a unrecognized parser is passed.

Reply:
Using an enum over a boolean is more future proof. It can be interesting if Babel adds a new language in the future or even if someone wants to disable the flow plugin. I'm all 👍

Reply:
Should I open a PR? 😅